### ISSUE TITLE
problem about burgers eqution

### ISSUE THREAD

**Issue by yonghcui** [lululxvi/deepxde#169]
hey lulu, i'm sorry to bother you, but i really have a problem. Well, when i ran the code Burgers.py which were written by M. Raissi, P. Perdikaris and G.E. Karniadakis, i know that it's not made by you, but i have noticed that the authors are in the same school and university as you, so i think maybe you will know how to solve the problem. The problem is that when i ran the code, i only can obtain the loss and error, training time , but no figures and the data of the result of burgers equation. I have seen that there is a plotting package in the code ,so i think the code maybe will create the figures and save them automatically, but i don't know why i cant obtain the figure. i think there maybe something wrong with my environment configuration since i don't know the tensorflow version and which package are needed for the code, so i wonder if you konw something about it? Thank you! Figure 1 is the result i obtained after running the code by phcharm, and figure 2 is the figure in the reference:"Physics Informed Deep Learning (Part I): Data-driven
Solutions of Nonlinear Partial Differential Equations" which i dont know why i cant obtain, and i also dont konw how to obtain the result of the soultion of the equation, and how to save them.
![1](https://user-images.githubusercontent.com/74175204/100685832-1688df80-33b8-11eb-9c14-efe6b79f25d1.PNG)
![2](https://user-images.githubusercontent.com/74175204/100685837-1b4d9380-33b8-11eb-9d9d-59a4e5d24346.PNG)

**Comment by lululxvi** [comment]
Sorry, I don't know how they plot the figure. There is the same example in DeepXDE: https://github.com/lululxvi/deepxde/blob/master/examples/Burgers.py

**Comment by yonghcui** [comment]
Thank you!
